### DISCUSSION TITLE
CtkOptionMenu showing with the default Tkinter OptionMenu dropdown style

### DISCUSSION THREAD

**Discussion by StardustD2** [TomSchimansky/CustomTkinter#1712] [General]
Unsure what might be causing it but I would love to get some help because it's really weird.
Below is a screenshot of the complex example from this repo completely unedited. Running the latest release versions of both Python and CustomTkinter.
![image](https://github.com/TomSchimansky/CustomTkinter/assets/117818418/651f797c-f3f0-4fff-bc18-75b94cab5e12)

**Comment by Akascape** [comment]
@StardustD2 
Here is a custom dropdown menu you can try: https://github.com/Akascape/CTkScrollableDropdown
![screenshots](https://user-images.githubusercontent.com/89206401/236677843-8d8b76fd-6145-47b1-8f4d-b6a64b08e1ea.png)

(This type of menu was removed from ctk because of some tkinter bugs appeared in different OS, specially in macosx ig. However, I have tried to solve them)

You can find more useful widgets like this here: https://github.com/Akascape/tkinter-toolkit

**Comment by StardustD2** [comment]
@Akascape 
I've actually been using that for a part of my app but it was a bit overkill for many elements. But I guess if it got removed then I have no other (easy) choice. Thanks though!

**Comment by Akascape** [comment]
@StardustD2 Wait, there is another type of dropdown menu I can make. But it will be a popup frame inside the main window instead of some popup toplevel.

**Comment by StardustD2** [comment]
@Akascape That would be amazing if you could! It would be absolutely perfect for my use case.

**Comment by Akascape** [comment]
@StardustD2 ok, I will notify you after making it.

**Comment by StardustD2** [comment]
@Akascape Wonderful, thank you so much!

**Comment by Akascape** [comment]
@StardustD2 Its done, please download the new version. Now you can replace the old dropdown with `CTkScrollableDropdownFrame`. Please let me know if there is any new issue.

**Comment by StardustD2** [comment]
@Akascape It's great. But I do have one quick question. Is there a way to hide the dropdowns if I click somewhere else on the main window? It's a bit finicky having to click on either the dropdown button or an option to hide it.

**Comment by Akascape** [comment]
@StardustD2 I added the tripple click and right button click for that.
